docs/node-mixin/dashboards/use.libsonnet

```diff
@@ -1,275 +1,460 @@
+    text: 'Prometheus',
+    value: 'Prometheus',
+  },
+  hide: 0,
+  label: null,
+  name: 'datasource',
+  options: [],
+  query: 'prometheus',
+  refresh: 1,
+  regex: '',
+  type: 'datasource',
+};
+
+local clusterTemplate =
+  template.new(
+    name='cluster',
+    datasource='$datasource',
+    query='label_values(node_cpu_seconds_total, %s)' % c._config.clusterLabel,
+    current='',
+    hide=if c._config.showMultiCluster then '' else '2',
+    refresh=2,
+    includeAll=false,
+    sort=1
+  );
+
+local CPUUtilisation =
+  graphPanel.new(
+    'CPU Utilisation',
+    datasource='$datasource',
+    span=6,
+    format='percentunit',
+    stack=true,
+    fill=10,
+    legend_show=false,
+  );
+
+local CPUSaturation =
+  // TODO: Is this a useful panel? At least there should be some explanation how load
+  // average relates to the "CPU saturation" in the title.
+  graphPanel.new(
+    'CPU Saturation (Load1 per CPU)',
+    datasource='$datasource',
+    span=6,
+    format='percentunit',
+    stack=true,
+    fill=10,
+    legend_show=false,
+  );
+
+local memoryUtilisation =
+  graphPanel.new(
+    'Memory Utilisation',
+    datasource='$datasource',
+    span=6,
+    format='percentunit',
+    stack=true,
+    fill=10,
+    legend_show=false,
+  );
+
+local memorySaturation =
+  graphPanel.new(
+    'Memory Saturation (Major Page Faults)',
+    datasource='$datasource',
+    span=6,
+    format='rds',
+    stack=true,
+    fill=10,
+    legend_show=false,
+  );
+
+local networkUtilisation =
+  graphPanel.new(
+    'Network Utilisation (Bytes Receive/Transmit)',
+    datasource='$datasource',
+    span=6,
+    format='Bps',
+    stack=true,
+    fill=10,
+    legend_show=false,
+  )
+  .addSeriesOverride({ alias: '/Receive/', stack: 'A' })
+  .addSeriesOverride({ alias: '/Transmit/', stack: 'B', transform: 'negative-Y' });
+
+local networkSaturation =
+  graphPanel.new(
+    'Network Saturation (Drops Receive/Transmit)',
+    datasource='$datasource',
+    span=6,
+    format='Bps',
+    stack=true,
+    fill=10,
+    legend_show=false,
+  )
+  .addSeriesOverride({ alias: '/ Receive/', stack: 'A' })
+  .addSeriesOverride({ alias: '/ Transmit/', stack: 'B', transform: 'negative-Y' });
+
+local diskIOUtilisation =
+  graphPanel.new(
+    'Disk IO Utilisation',
+    datasource='$datasource',
+    span=6,
+    format='percentunit',
+    stack=true,
+    fill=10,
+    legend_show=false,
+  );
+
+local diskIOSaturation =
+  graphPanel.new(
+    'Disk IO Saturation',
+    datasource='$datasource',
+    span=6,
+    format='percentunit',
+    stack=true,
+    fill=10,
+    legend_show=false,
+  );
+
+local diskSpaceUtilisation =
+  graphPanel.new(
+    'Disk Space Utilisation',
+    datasource='$datasource',
+    span=12,
+    format='percentunit',
+    stack=false,
+    fill=5,
+    legend_show=true,
+    legend_alignAsTable=true,
+    legend_current=true,
+    legend_avg=true,
+    legend_rightSide=true,
+    legend_sortDesc=true,
+  );
```

Comment:
Hmmm, I see your point, but at the same time, I think that stacking disk usage doesn't make much sense for cluster and multicluster views if we are using ratios.

With several clusters/nodes, it is easy to surpass 100% utilization when stacking. In my opinion that is more confusing.

Would it make sense if we change the query to show utilization in bytes instead of ratio?

Reply:
By the way, I don't have really strong feelings about my comment above. No problem stacking everything if you folks prefer